src/app/cart/cart-item/cart-item.component.html

```diff
@@ -0,0 +1,12 @@
+<li class="cart--item-list">
+  <img
+    class="cart--item-icon"
+    src="../../../assets/icons/{{ cartItem.id }}.svg"
```

Comment:
We shouldn't need to do relative paths like this. We could just do `"assets/icons{{cartItemid}}.svg"`